packages/ui/components/profile/ProfileHomeCard.tsx

```diff
@@ -0,0 +1,117 @@
+import { useRouter } from 'next/router'
+import { useTranslation } from 'react-i18next'
+
+import { ArrowOutward, Layers, Payments } from '@dao-dao/icons'
+
+import { ButtonLink } from '../Button'
+import { IconButton } from '../IconButton'
+import { ProfileCardWrapper } from './ProfileCardWrapper'
+
+export interface ProfileHomeCardProps {
+  walletName: string
+  profileImgUrl: string
+  established: Date
+  tokenSymbol: string
+  unstakedBalance: number
+  stakedBalance: number
+  numDaos: number
+  numVotes: number
+  inboxProposalCount: number
+}
+
+export const ProfileHomeCard = ({
+  walletName,
+  profileImgUrl,
+  established,
+  tokenSymbol,
+  unstakedBalance,
+  stakedBalance,
+  numDaos,
+  numVotes,
+  inboxProposalCount,
+}: ProfileHomeCardProps) => {
+  const { t } = useTranslation()
+  const router = useRouter()
+
+  return (
+    <ProfileCardWrapper
+      childContainerClassName="p-0"
+      established={established}
+      imgUrl={profileImgUrl}
+      underHeaderComponent={
+        <div className="grid grid-cols-[1fr_1px_1fr] gap-2 justify-items-center items-center self-stretch mt-3">
+          <div className="flex flex-col items-stretch text-center">
+            <Payments className="self-center mb-4 w-5 h-4 text-center text-icon-secondary" />
+            <p className="mb-1 secondary-text">{t('title.holdings')}</p>
+            <p className="font-mono break-words title-text">
+              {unstakedBalance.toLocaleString(undefined, {
+                // eslint-disable-next-line i18next/no-literal-string
+                notation: 'compact',
+                maximumFractionDigits: 2,
+              })}{' '}
+              ${tokenSymbol}
```

Comment:
i created https://github.com/DA0-DA0/dao-dao-ui/blob/03a7e31cc80c76bfede3bbdf190bf20a76aece10/packages/i18n/locales/en/translation.json#L199 lmk whether u think that's overkill in this sitation

Reply:
The important part of this is the `compact` and `maximumFractionDigits` that control its formatting. I don't know if the i18n library lets us automatically do compact notation (suffixing with K, M, B, or T automatically as the number increases), but if it is I'm down to make a new translation key :)

This is a very common pattern and either should be a translation key or a util function. I'm kinda thinking a util function because nothing about this needs to be translated. wdyt?